server/src/services/merchants.ts

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
+import {MerchantPayload} from '../interfaces';
 import {merchantStorage} from '../storage';
 
-const getAllMerchants = async () => await merchantStorage.getAllMerchants();
+const addMerchant = async (merchant: MerchantPayload) =>
+  await merchantStorage.addMerchant(merchant);
```

Comment:
```suggestion
  merchantStorage.addMerchant(merchant);
```
Actually you won't need await here!
Read: https://eslint.org/docs/rules/no-return-await

Reply:
Oh! Because the arrow notation has implicit return right?

Reply:
Yep arrow function w/ the braces is an implicit return so over here what's going on now is equivalent to a `return await`!